jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java

```diff
@@ -201,7 +204,15 @@ public void init() {
         initGridColumns();
         initDateTimePicker();
 
+        Button button = new Button(Constants.INSTANCE.Add_Parameter(), new ClickHandler() {
```

Comment:
Please use _GWT.create_ instead, as this is needed for GWTMockito to work properly.
I would suggest something like:
`Button button = GWT.create(Button.class);`
 `button.setText(Constants.INSTANCE.Add_Parameter());`
`button.addClickHandler(e -> addNewParameter());`

Reply:
Ok